bundle/src/main/java/com/adobe/acs/commons/cloudservices/pwa/impl/PwaManifestServlet.java

```diff
@@ -44,17 +44,17 @@
     @Reference
     private ModelFactory modelFactory;
 
- 
+
     @Reference
     private ResourceResolverFactory resourceResolverFactory;
- 
- 
+
+
     @Override
     protected final void doGet(SlingHttpServletRequest request, SlingHttpServletResponse response) throws ServletException, IOException {
         response.setContentType("application/x-web-app-manifest+json; charset=UTF-8");
         response.setCharacterEncoding("UTF-8");
 
- 
+
         ResourceResolver serviceResourceResolver = null;
         try {
             serviceResourceResolver = resourceResolverFactory.getServiceResourceResolver(AUTH_INFO);
```

Comment:
The sling models can’t cleanly close open service resolvers in header.html/footer.html so we need ACLs regardless

Reply:
do sling models need to ? they depend upon a request object and are passed by the caller . We can discuss. I don't see memory leak though

Reply:
yes, exactly. we need ACLs then.